x-pack/platform/packages/shared/kbn-elastic-assistant-common/impl/schemas/prompts/bulk_crud_prompts_route.schema.yaml

```diff
@@ -9,7 +9,10 @@ paths:
       x-labels: [ess, serverless]
       operationId: PerformPromptsBulkAction
       summary: Apply a bulk action to prompts
-      description: Apply a bulk action to multiple prompts. The bulk action is applied to all prompts that match the filter or to the list of prompts by their IDs.
+      description: >
+        Apply a bulk action to multiple prompts. The bulk action is applied to all prompts
+        that match the filter or to the list of prompts by their IDs. This action allows
+        for bulk create, update, or delete operations.
```

Comment:
Does the `>` provide some additional styling?

<img width="543" alt="image" src="https://github.com/user-attachments/assets/b5008868-d396-4b2b-9541-e2f598e95731" />

Reply:
Yeah must be a space, im taking it out in my latest commit. This is something our robot friend added